fatiando/mesher/geometry.py

```diff
@@ -72,9 +72,11 @@ class Polygon(GeometricElement):
 
     """
 
-    def __init__(self, vertices, props=None):
+    def __init__(self, vertices, props=None, force_clockwise=False):
```

Comment:
It's probably better to default this to `True` since having counterclockwise vertices is usually a bug.